endless-sky.appdata.xml

```diff
@@ -24,13 +24,19 @@
   <url type="homepage">https://endless-sky.github.io/</url>
   <screenshots>
     <screenshot type="default">
-      <image>https://endless-sky.github.io/screenshots/fight.jpg</image>
+      <image>https://endless-sky.github.io/images/screenshots/battle.jpg</image>
     </screenshot>
-    <screenshot>
-      <image>https://endless-sky.github.io/screenshots/hyperspace.jpg</image>
+	<screenshot>
```

Comment:
There's a tab here instead of spaces (and there are 2 more like it below).

Reply:
Oh darn, my bad. Will fix soon.